x-pack/plugins/fleet/scripts/install_all_packages/install_all_packages.ts

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import fetch from 'node-fetch';
+import { ToolingLog } from '@kbn/dev-utils';
+import ReaddPackage from 'read-pkg';
+
+const REGISTRY_URL = 'https://epr-snapshot.elastic.co';
+const KIBANA_URL = 'http://localhost:5601';
+const KIBANA_USERNAME = 'elastic';
+const KIBANA_PASSWORD = 'changeme';
+
+const KIBANA_VERSION = ReaddPackage.sync().version;
+
+const SKIP_PACKAGES: string[] = [];
+
+async function installPackage(name: string, version: string) {
+  const start = Date.now();
+  const res = await fetch(`${KIBANA_URL}/api/fleet/epm/packages/${name}-${version}`, {
+    headers: {
+      accept: '*/*',
+      'content-type': 'application/json',
+      'kbn-xsrf': 'xyz',
+      Authorization:
+        'Basic ' + Buffer.from(`${KIBANA_USERNAME}:${KIBANA_PASSWORD}`).toString('base64'),
+    },
+    body: JSON.stringify({ force: true }),
+    method: 'POST',
+  });
+  const end = Date.now();
+
+  const body = await res.json();
+
+  return { body, status: res.status, took: (end - start) / 1000 };
+}
+
+async function deletePackage(name: string, version: string) {
+  const res = await fetch(`${KIBANA_URL}/api/fleet/epm/packages/${name}-${version}`, {
+    headers: {
+      accept: '*/*',
+      'content-type': 'application/json',
+      'kbn-xsrf': 'xyz',
+      Authorization:
+        'Basic ' + Buffer.from(`${KIBANA_USERNAME}:${KIBANA_PASSWORD}`).toString('base64'),
+    },
+    method: 'DELETE',
+  });
+
+  const body = await res.json();
+
+  return { body, status: res.status };
+}
+
+async function getAllPackages() {
+  const res = await fetch(
+    `${REGISTRY_URL}/search?experimental=true&kibana.version=${KIBANA_VERSION}`,
+    {
+      headers: {
+        accept:
```

Comment:
this accept string may not be needed I think I just got it from "copy as fetch" in chrome :D